flycheck.el

```diff
@@ -4288,6 +4304,10 @@ symbols in the command."
           ;; example for such a conflict.
           (setq process (apply 'start-process (format "flycheck-%s" checker)
                                nil program args))
+          (when (and (flycheck-checker-standard-input-p checker)
+                     (process-live-p process))
+            (process-send-region process (point-min) (point-max))
```

Comment:
Ok, since `process-send-region` checks for boundaries, you'll need to use the following instead.

```
(save-restriction
  (widen)
  (process-send-region (point-min) (point-max)))
```